jenkins/pipeline-fb.groovy

```diff
@@ -0,0 +1,229 @@
+pipeline_timeout = 10
+
+if (
+    (params.ANALYZER_OPTS.contains('-DWITH_ASAN=ON')) ||
+    (params.ANALYZER_OPTS.contains('-DWITH_UBSAN=ON'))
+    ) { pipeline_timeout = 24 }
+
+if (params.ANALYZER_OPTS.contains('-DWITH_VALGRIND=ON'))
+    { pipeline_timeout = 144 }
+
+if (
+    ((params.ANALYZER_OPTS.contains('-DWITH_ASAN=ON')) &&
+    (params.ANALYZER_OPTS.contains('-DWITH_ASAN_SCOPE=ON')) &&
+    (params.ANALYZER_OPTS.contains('-DWITH_UBSAN=ON'))) ||
+    ((params.MTR_ARGS.contains('--big-test')) || (params.MTR_ARGS.contains('--only-big-test')))
+    ) {
+        LABEL = 'docker-32gb'
```

Comment:
This `LABEL = 'docker-32gb'` is probably not needed here as always use `docker-32gb`
```suggestion
```

Reply:
yes, you are right we can remove it